lib/metaractor/parameters.rb

```diff
@@ -143,7 +143,7 @@ def remove_blank_values
 
     def apply_defaults
       parameters.each do |name, parameter|
-        next unless parameter[:default]
+        next unless parameter.has_key?(:default)
```

Comment:
Shoot. `parameter` doesn't respond to `has_key?`

Reply:
It also appears that the failure reporting in buildkite isn't working for some reason.

Reply:
Ha yeah, I realized that after I pushed it up. I tried to get it working my docker env, but it wanted access to my ssh keys and my `.gem` directory. First one, why? Second one, doesn't exist. All of my ruby stuff runs inside of docker always.

Reply:
That's the `release` container. You don't need to run that one.